src/sqlancer/Main.java

```diff
@@ -211,6 +243,13 @@ private void printState(FileWriter writer, StateToReproduce state) {
             }
         }
 
+        private String processQueryPlan(String queryPlan) {
```

Comment:
Could we use a more insightful name here, like `removeNamesFromQueryPlans` or so?